e2e/cypress/integration/channel/close_direct_group_spec.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod @smoke
+// Group: @channel @channel_settings
```

Comment:
@saturninoabril I just copied these lines from another file, but I'm not sure what they mean. Is there any place with docs on what to put here?

Reply:
Thanks for checking. These are what we called test metadata ([link](https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#adding-test-metadata-on-spec-files)). You may remain the "Group" as is. For "Stage", please remove. Once merged, QA team will monitor this spec and will promote to `@prod` if we found the test to be stable.

Reply:
Thank you! Done in https://github.com/mattermost/mattermost-webapp/pull/5422/commits/bac1b771a1f226ad2bc58a6c09068687cebd1b3c